google-cloud-firestore/src/main/java/com/google/cloud/firestore/Query.java

```diff
@@ -810,7 +810,15 @@ public Query endAt(@Nonnull DocumentSnapshot snapshot) {
     structuredQuery.addFrom(
         StructuredQuery.CollectionSelector.newBuilder().setCollectionId(path.getId()));
 
-    if (!options.fieldFilters.isEmpty()) {
+    if (options.fieldFilters.size() == 1) {
+      Filter filter = options.fieldFilters.get(0).toProto();
+      if (filter.hasFieldFilter()){
+        structuredQuery.getWhereBuilder().setFieldFilter(filter.getFieldFilter());
+      } else {
+        Preconditions.checkState (filter.hasUnaryFilter(), "Expected a UnaryFilter or a FieldFilter.");
+        structuredQuery.getWhereBuilder().setUnaryFilter(filter.getUnaryFilter());
```

Comment:
This block is formatted somewhat strangely, could you try google-java-format this?
[This script](https://github.com/google/google-java-format/blob/master/scripts/google-java-format-diff.py) also helps only formatting things you changed.

LGTM otherwise

Reply:
Re-run 1.5 of google-java-format on the codebase.